### PR TITLE
Helm: Prometheus exporter

### PR DESCRIPTION
**Pull request by hunter**
Adds basic support for the digitalocean ceph-exporter to allow metrics
collection

### CODE REVIEW

File: examples/helm/ceph/templates/exporter/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      containers:
+      - name: ceph-exporter
+        image: {{ .Values.images.exporter }}
+        imagePullPolicy: {{ .Values.image_policy.pull }}
+        ports:
+          - name: ceph-exporter
+            containerPort: {{ .Values.service.exporter.port }}
+        volumeMounts:
+          - name: ceph-conf
+            mountPath: /etc/ceph/ceph.conf
```
Comment (leseb) [inline examples/helm/ceph/templates/exporter/deployment.yaml]: Do we have a way to expose a different cluster name?

Comment (hunter) [inline examples/helm/ceph/templates/exporter/deployment.yaml]: Possible but may need some rethinking across all of the resources in the helm chart. Something that can be done in another PR?

Comment (leseb) [inline examples/helm/ceph/templates/exporter/deployment.yaml]: Absolutely.

**Comment by leseb** [comment]
I think something went wrong during your rebase, can you rebase again against master?

**Pull request merged**